repo_checker.py

```diff
@@ -119,6 +121,9 @@ def prepare_review(self):
                 self.logger.debug('{}: not staged'.format(request.reqid))
                 continue
 
+            if self.limit_group and group != 'openSUSE:Leap:15.0:Staging:C':
```

Comment:
Assume this is not intentional to have hard-coded group?

Reply:
oh, shit :)